src/bin/client-demo.rs

```diff
@@ -243,13 +240,13 @@ fn main() {
     println!("Got last ID {:?}", last_id);
 
     let mut seed = [0u8; 32];
-    seed.copy_from_slice(&demo.mint.keypair().public_key_bytes()[..32]);
+    seed.copy_from_slice(&mint.keypair().public_key_bytes()[..32]);
     let rnd = GenKeys::new(seed);
 
     println!("Creating keypairs...");
-    let txs = demo.num_accounts / 2;
-    let keypairs = rnd.gen_n_keypairs(demo.num_accounts);
-    let keypair_pairs: Vec<_> = keypairs.chunks(2).collect();
+    let num_accounts = 500_000;
+    let txs = num_accounts / 2;
```

Comment:
I think txs should equal num_accounts now

Reply:
Oh yeah, good point.